styles/arch-wiki/catppuccin.user.css

```diff
@@ -16,17 +16,24 @@
 ==/UserStyle== */
 
 @-moz-document domain('wiki.archlinux.org') {
-  @media (prefers-color-scheme: dark) {
+  @media (prefers-color-scheme: light) {
     :root {
-      #catppuccin(@darkFlavor, @accentColor);
+      #catppuccin(@lightFlavor, @accentColor);
     }
   }
-  @media (prefers-color-scheme: light) {
+  @media (prefers-color-scheme: dark) {
     :root {
-      #catppuccin(@lightFlavor, @accentColor);
+      #catppuccin(@darkFlavor, @accentColor);
     }
   }
 
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
```

Comment:
Please remove this boilerplate as I'm guessing it doesn't do anything.

Reply:
alright